templates/layouts/roq-default/main.html

```diff
@@ -9,8 +9,13 @@
   .menu li .fa-info {
     padding-left: 4px;
   }
+  .search-button {
+    margin-left: 15px;
+    background: none;
+  }
```

Comment:
Here you can add:
```
    color: #515151;
    display: block;
    font-size: 18px;
    transition: all .35s cubic-bezier(.13,.43,.54,1.82);
```

and when hovered:
```
.search-button:hover {
    color: #333030;
}
```

Reply:
It seems the font size slightly change when hovered too :thinking:

Reply:
> It seems the font size slightly change when hovered too 🤔

yes there's:
```
.menu ul li a:hover i {
  transform: scale(1.1);
}
```

I'll do it